internal/helpers/env.go

```diff
@@ -2,11 +2,14 @@ package helpers
 
 import "os"
 
-// GetEnv retrieves an environment variable with a default value
+// GetEnv retrieves list of possible environment variables with a default value
 // if the environment variable is missing
-func GetEnv(key, fallback string) string {
-	if value, ok := os.LookupEnv(key); ok {
-		return value
+func GetEnv(keys_and_fallback ...string) string {
```

Comment:
```suggestion
func GetEnv(keysAndFallback ...string) string {
```
How do you feel about using camelCase here?